docs/Development/CONTRIBUTING.md

```diff
@@ -10,6 +10,14 @@ Please note we have a code of conduct, please follow it in all your interactions
 
 2.Update the `README.md` file with details of changes to the interface, this includes new environment variables, exposed ports, useful file locations and container parameters.
```

Comment:
environment variables?
exposed ports?
Are we really using these stuff?

Reply:
so, should I delete this whole point or only change the stuff that you are indicating?

Reply:
ask yourself this question!

Reply:
Are you on it?

Reply:
yes

Reply:
Complete it fast!
you are taking a lot of time on this fix.